### PR TITLE
Added ssl verify parameter into request

### PR DESCRIPTION
**Pull request by lukas-bednar**
Fix #72

### CODE REVIEW

File: pytest_jira.py [hunk 1]
```diff
@@ -162,6 +162,8 @@ def __init__(
             self.basic_auth = None
 
     def _jira_request(self, url, method='get', **kwargs):
+        if 'verify' not in kwargs:
+            kwargs['verify'] = self.verify
```
Comment (lukas-bednar) [inline pytest_jira.py]: I positive case it should be ``/path/to/cert`` not ``True`` as it is now ...

Comment (liiight) [inline pytest_jira.py]: Cool, I wasn't aware that requests supports this natively or I would have added it myself

Comment (liiight) [inline pytest_jira.py]: So maybe:
```python 
if self.verify is False:
    kwargs['verify'] = False
```

Comment (lukas-bednar) [inline pytest_jira.py]: I will have to handle it properly, problem is on L320, where it is load as boolean.

Comment (lukas-bednar) [inline pytest_jira.py]: After closer investigation, it accepts true as well. in such case it uses system configuration.
So I think it is OK.

**Comment by liiight** [comment]
Cool. So when are you releasing?

**Comment by lukas-bednar** [comment]
> Cool. So when are you releasing?

I think that I will do it by EOW, I want to try move on with that info on skipped issues  ...

**Comment by liiight** [comment]
Thanks for this (and for everything else)!

**Pull request merged**